frontend/webpack.config.ts

```diff
@@ -19,7 +19,11 @@ interface Configuration extends webpack.Configuration {
 const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
-const { SUPPORTED_LOCALES } = require('./public/i18next-parser.config');
+
+const SUPPORTED_LOCALES = {
+  en: 'English',
+  ja: '日本語',
+};
```

Comment:
Don't initial this as a map and format later. Better to just specify it inline in the format we need.